docs/decisions/0002-text-editor-adr.md

```diff
@@ -0,0 +1,86 @@
+# Status
+
+Accepted
+
+# Context
+
+We seek to hoist the legacy text (HTML) text editor out of the monolith into a decoupled React application
+without reducing editor capabilities in the process.
+
+The legacy editor delegates the bulk of its operations to the tinyMCE editor
+
+
+
+# Decision set (each decision small)
+
+1. Rely on editor-wide Blockstore networking utilities for loading from, and saving to, XBlock content
+2. Continue using the tinyMCE editor
+3. Wrap the tinyMCE editor so as to be able to handle it as a vanilla React component
+4. Use a mix of configuration and tinyMCE API calls to customize the editor as needed 
```

Comment:
We haven't defined anywhere what is meant by tinyMCE API. Readers could be confused by this term. A worthy defintion is perhaps the api tinyMCE provides to perform operations on the editor object and its state.

Reply:
TinyMCE had been mentioned earlier, in the Context section. But it hadn't been defined. I included a link to TinyMCE documentation, as well as the tag line used by TinyMCE to define itself.